lib/devicetrust/authn/authn_test.go

```diff
@@ -61,38 +62,34 @@ func TestCeremony_Run(t *testing.T) {
 	}
 
 	tests := []struct {
-		name  string
-		dev   testenv.FakeDevice
-		certs *devicepb.UserCertificates
+		name      string
+		dev       testenv.FakeDevice
+		certs     *devicepb.UserCertificates
+		sshCert   []byte
```

Comment:
I think we only need one of these.

Reply:
yeah these were all the same for each test so i pulled it all out